prow/report/report.go

```diff
@@ -76,7 +76,11 @@ func reportStatus(ghc GithubClient, pj v1.ProwJob, childDescription string) erro
 		if err != nil {
 			return err
 		}
-		if err := ghc.CreateStatus(refs.Org, refs.Repo, refs.Pulls[0].SHA, github.Status{
+		SHA := refs.BaseSHA
```

Comment:
wouldn't sha be more appropriate?

Reply:
I thought SHA is a proper noun in English?

Reply:
Not sure what you mean. Looking it up, sha is an abbreviation for "secure hash algorithm."

a) no, sha standard for secure hash algorithm.
b) we'd say url here, not URL
c) we'd say frog here, not Frog or FROG.

my point is that this isn't an exported field, so should be lowercase.

Reply:
done